lib/rubocop/cop/rspec/subject_stub.rb

```diff
@@ -72,7 +72,7 @@ class SubjectStub < Base
         def_node_matcher :subject?, <<-PATTERN
           (block
             (send nil?
-              {:subject (sym $_) | $:subject}
+              {:subject (sym $_) | :subject! (sym $_) | $:subject | $:subject! }
```

Comment:
How about the following?
```suggestion
              { #Subjects.all (sym $_) | $#Subjects.all }
```

Reply:
Oops. Something is wrong with the suggestions 🤔.

```ruby
        def_node_matcher :subject?, <<-PATTERN
          (block
            (send nil?
              { #Subjects.all (sym $_) | $#Subjects.all }
            ) args ...)
        PATTERN
```

Reply:
Done!